app/views/course/material/folders/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header(@folder.name)
+= page_header(display_name(@folder))
```

Comment:
I greped, this seems to be the only occurrence. and sorry, why I want to link to the folder ?